### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by mluhovyi**
Updated old packages, fixed integration test run, added integration test instruction, updated workflow

### CODE REVIEW

File: test/integration-tests/test-utilities.ts [hunk 1]
```diff
@@ -40,14 +40,14 @@ async function createTempDir(tempDir: string, folderName: string) {
...
 function slsDeploy(tempDir: string): Promise<string> {
-  return exec(`cd ${tempDir} && $(npm bin)/serverless deploy`);
+  return exec(`cd ${tempDir} && serverless deploy`);
 }
 
 /**
  * Runs `sls remove` for the given folder
  */
 function slsRemove(tempDir: string): Promise<string> {
-  return exec(`cd ${tempDir} && $(npm bin)/serverless remove`);
```
Comment (rddimon) [inline test/integration-tests/test-utilities.ts:50]: Why is that removed?

**Comment by rddimon** [comment]
Could you please also update `CHANGELOG.md`?

File: test/integration-tests/test-utilities.ts (hunk 1 above)
Comment (mluhovyi) [reply test/integration-tests/test-utilities.ts:50]: $(npm bin) is not needed here and limits systems where this command can run

Comment (rddimon) [reply test/integration-tests/test-utilities.ts:50]: `$(npm bin)` will use local packages without that command some of the global packages can be involved
Could you please revert it back?

**Review by rddimon — APPROVED** [review]
